### ISSUE TITLE
DEM

### ISSUE THREAD

**Issue by lllxllll** [cfandel/gottesacker#2]
Can we use the DEM to control the extent of  3D geological model ,for example, using DEM to cut the model.

**Comment by cfandel** [comment]
Yes, you can do this with the crop2topo() and crop2raster() functions in the mapping module here: https://github.com/cfandel/gottesacker/blob/master/mapping.py

**Comment by lllxllll** [comment]
I cut the lith block use crop2topo() ,but  I encountered some problems when I use the cropped lith block to  compute the vertices and simplices, because the function gp.get_surfaces() require float number, but the cropped lith block contains nan ,I don't quite understand how to solve this problem to get a corpped 3D model.

**Comment by cfandel** [comment]
Yes, the way that cropping works is that it replaces all the cells outside the raster you provide with nans. 
You should do all the gempy operations you want to do first, and then crop the output from gempy as the very last step.
